app/src/main/java/org/mozilla/fenix/library/bookmarks/edit/EditBookmarkFragment.kt

```diff
@@ -113,14 +114,15 @@ class EditBookmarkFragment : Fragment(R.layout.fragment_edit_bookmark) {
 
     private fun initToolbar() {
         val activity = activity as? AppCompatActivity
-        val toolbar = activity?.findViewById<Toolbar>(R.id.navigationToolbar)
+        (activity as HomeActivity).getSupportActionBarAndInflateIfNecessary()
+        val toolbar = activity.findViewById<Toolbar>(R.id.navigationToolbar)
         context?.let {
             toolbar?.setToolbarColors(
                 foreground = it.getColorFromAttr(R.attr.primaryText),
                 background = it.getColorFromAttr(R.attr.foundation)
             )
         }
-        activity?.supportActionBar?.show()
+        activity.supportActionBar!!.show()
```

Comment:
It's my opinion that we should redirect all callers to use `getSupportActionBar...` instead of calling `activity.supportActionBar`. We should follow that behavior ourselves here too! We can store the actionBar in a local variable above and re-use it here.